components/constellation/constellation.rs

```diff
@@ -1053,6 +1061,10 @@ impl<Message, LTF, STF> Constellation<Message, LTF, STF>
             warn!("Exit resource thread failed ({})", e);
         }
 
+        if let Some(ref chan) = self.debugger_chan {
+            shutdown_server(chan);
```

Comment:
nit: this would be slightly more clear as `debugger::shutdown_server(chan)` I think.